.github/workflows/changelog.yml

```diff
@@ -0,0 +1,69 @@
+name: Generate changelog
+
+on:
+  workflow_dispatch:
+    inputs:
+
+      repository:
+        description: "Repository in which the changelog will be added."
+        required: false
```

Comment:
We should make this required as the default text box can be emptied. We should always require a repository so there is no ambiguity on where this will be applied.